analysis_options.yaml

```diff
@@ -1,28 +1,20 @@
-# This file configures the analyzer, which statically analyzes Dart code to
-# check for errors, warnings, and lints.
-#
-# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
-# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
-# invoked from the command line by running `flutter analyze`.
-
-# The following line activates a set of recommended lints for Flutter apps,
-# packages, and plugins designed to encourage good coding practices.
 include: package:flutter_lints/flutter.yaml
 
+analyzer:
+  exclude: [build/**]
+  errors:
+    always_use_package_imports: error
+    directives_ordering: error
+    prefer_single_quotes: error
+    require_trailing_commas: error
```

Comment:
It seems like every rule has an default severity level set and it is not possible to set every rule to `error`. We might want to add more rules defined in `flutter_lints` later.